src/urllib3/exceptions.py

```diff
@@ -305,7 +305,8 @@ class IncompleteRead(HTTPError, httplib_IncompleteRead):
     expected: int
 
     def __init__(self, partial: int, expected: int) -> None:
-        super().__init__(partial, expected)
```

Comment:
`Argument 1 to "__init__" of "IncompleteRead" has incompatible type "int"; expected "bytes"`

Reply:
Huh, `partial` is definitely supposed to be `bytes` but we're using it with `int`?